gp2gp-translator/src/integrationTest/java/uk/nhs/adaptors/pss/translator/amqp/ServiceFailureIT.java

```diff
@@ -197,39 +203,62 @@ public void When_ReceivingEhrExtract_WithMhsWebClientRequestException_Expect_Mig
             .doCallRealMethod()
             .when(mhsClientService).send(any());
 
-        sendInboundMessageToQueue("/json/LargeMessage/Scenario_3/uk06.json");
+        sendInboundMessageToQueue(JSON_LARGE_MESSAGE_SCENARIO_3_UK_06_JSON);
 
         await().atMost(Duration.ofMinutes(TWO_MINUTES_LONG))
             .until(this::hasContinueMessageBeenReceived);
 
-        sendInboundMessageToQueue("/json/LargeMessage/Scenario_3/copc.json");
+        sendInboundMessageToQueue(JSON_LARGE_MESSAGE_SCENARIO_3_COPC_JSON);
 
         await().until(this::isEhrMigrationCompleted);
 
-        verifyBundle("/json/LargeMessage/expectedBundleScenario3.json");
+        verifyBundle(JSON_LARGE_MESSAGE_EXPECTED_BUNDLE_SCENARIO_3_JSON);
     }
 
     @Test
     public void When_ReceivingCopc_WithMhsWebClientRequestException_Expect_MigrationCompletesWhenMhsRecovers() throws JSONException {
 
-        sendInboundMessageToQueue("/json/LargeMessage/Scenario_3/uk06.json");
+        sendInboundMessageToQueue(JSON_LARGE_MESSAGE_SCENARIO_3_UK_06_JSON);
 
-        await().until(this::hasContinueMessageBeenReceived);
+        await().atMost(Duration.ofMinutes(TWO_MINUTES_LONG)).until(this::hasContinueMessageBeenReceived);
```

Comment:
Similarly, should this be extended? I can't see any reason why it needs to be. 🤔 If not, it'll just make the tests slower to fail.